test_unittest.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from input import function
+
+class Test(unittest.TestCase):

```

Comment:
We should probably give this test class a better name:
```suggestion
class TestUserInput(unittest.TestCase):
```
A decent example I found quickly that shows python naming conventions is here https://stackoverflow.com/questions/32672551/python-unittesting-for-a-class-method

Reply:
Similarly, your file names should reflect what things are:

Change `input.py` to something like `user_input_utils.py` and change `test_unittest.py` to `test_user_input.py`